app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -19,29 +19,29 @@
 /**
  * Simple login activity class.
  */
-public final class LoginActivity extends Activity {
+public final class LoginActivity extends AppCompatActivity {
```

Comment:
I think consistency across apps within the Android ecosystem is more important then (what you consider) a nicer title. I think the designers over at Android are pretty good authorities on what looks good and how you should design your app...

But it is kinda offtopic for this PR, so we will one day figure that out in another issue/PR 😄